keps/sig-storage/20200427-csi-driver-skip-permission.md

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
There is a new KEP template structure and a production readiness questionaire that needs to be filled out.

Reply:
I've updated the KEP to hopefully adhere to the new standard.